services/hanke-service/src/test/resources/sql/senaatintorin-hanke.sql

```diff
@@ -58,4 +59,5 @@ VALUES (23,
         0,
         0,
         15,
-        5);
+        5,
+        'Hankealue 1');
```

Comment:
Maybe a bit more personality for this one?
```suggestion
        'Senaatintori');
```

Reply:
Changed to "Senaatintori"